Automattic/wp-calypso
# Stats: Add static page modules show/hide settings

Related to #77718 

## Proposed Changes

* Add modules settings panel with the `Popover` component.
* Gate the settings panel with the feature flag `stats/modules-settings`.

## Testing Instructions



* Spin this change with the live branch link.
* Navigate to Stats > `Traffic` page.
* Append feature flag to the URL: `?flags=stats/modules-settings`.
* Ensure the settings button and panel are in line with the design.
* Ensure the toggle buttons work without changing the display of modules yet.

<img width="1303" alt="截圖 2023-06-04 下午10 18 31" src="https://github.com/Automattic/wp-calypso/assets/6869813/db755649-853a-4203-a19b-9911096e44d9">

## Pre-merge Checklist
